shared/src/main/scala/Elements.scala

```diff
@@ -477,6 +477,20 @@ object Elements:
         ListHelpers.map(a, ListHelpers.makeIterable(b, Some(true)))
     }
 
+    val merge: Dyad = addElem(
+      Dyad,
+      "J",
+      "Merge",
+      List("merge"),
+      "a: lst, b: lst -> Merge a and b",
+    ) {
+      case (a: VNum, b: VNum)   => MiscHelpers.eval(a.toString + b.toString)
+      case (a: VVal, b: VVal)   => MiscHelpers.add(a, b)
```

Comment:
```suggestion
      case (a: String, b: String)   => MiscHelpers.add(a, b)
```
Since `(VNum, VNum)` has already been matched on. Are we sure we don't want to overload this to something else? String concatenation already exists elsewhere

Reply:
This is for the `VNum`, `String` case as well though. It's simply doing version 2 behaviour

Reply:
Oh, I see.